spec/chromium-spec.js

```diff
@@ -937,6 +937,23 @@ describe('chromium feature', () => {
   })
 
   describe('storage', () => {
+    describe('DOM storage quota override', () => {
+      ['localStorage', 'sessionStorage'].forEach((storageName) => {
+        it(`can store at least 50MiB in ${storageName}`, (done) => {
+          assert.doesNotThrow(() => {
```

Comment:
`assert.doesNotThrow` is not needed here, please remove it.
The whole test kind of serves as an expectation that its body doesn't throw, it will fail if there are any exceptions.

Reply:
Oh, duh, yes. I think I wrote that originally because I didn't realize there was anything else to `assert`, but then I added a check for length.